### ISSUE TITLE
Anti-CA6

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#7235]
**Preferred term label:**
anti-carbonic anhydrase VI antibody positivity

**Synonyms**
Anti-CA6 antibody positivity

**Definition (free text, please give PubMed ID)**
PMID: 34249010

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:289390

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by pnrobinson** [comment]
added
